Maple2.Server.Game/Packets/FishingPacket.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Collections.Generic;
+using System.Numerics;
+using Maple2.Model.Common;
+using Maple2.Model.Enum;
+using Maple2.Model.Error;
+using Maple2.Model.Game;
+using Maple2.Model.Metadata;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.Packets;
+using Maple2.Tools.Extensions;
+
+namespace Maple2.Server.Game.Packets;
+
+public static class FishingPacket {
+    private enum Command : byte {
```

Comment:
There's actually 2 more ops: 10, 11 if you want to take a look
``` python
elif f == 10: # s_fishing_simulate
    count = add_int("count")
    for _ in range(count):
        add_int("unknown")
        add_int("unknown")
        count = add_int("unknown")
        for _ in range(count):
            add_int("unknown")
elif f == 11:
    b = add_bool("start")
    if b:
        pass # client sends fishing start (op 9)
```

I think op 11 might have something to do with autofishing? That bool is the same as the empty byte in op 8 and it might tell the client to keep fishing after catching?

Reply:
Is this flipped around? 10 seems to be 11 and vice versa

Reply:
yea my bad, mixed them up